private.php

```diff
@@ -1688,16 +1689,21 @@
 		$mybb->input['check'] = $mybb->get_input('check', MyBB::INPUT_ARRAY);
```

Comment:
```suggestion
		$pms = array_map('intval', array_keys($mybb->get_input('check', MyBB::INPUT_ARRAY)));

```

Reply:
If we are taking filtered PM IDs in `$pms` shouldn't we check this variable's emptiness in next line?
In place of `if(!empty($mybb->input['check']))`?

Why don't we overwrite same var as it was? Why introduce a new var?

Reply:
Yes should probably change the `if` but it's functionally equivalent. Just made a new variable because it is representing in a different way to prevent any possible confusion.
`$pms` is an array of PM IDs, `$mybb->input['check']` has the PM IDs as keys and the values as `1`.